drivers/windows/pandaJ2534DLL/PandaJ2534Device.cpp

```diff
@@ -8,11 +8,15 @@ PandaJ2534Device::PandaJ2534Device(std::unique_ptr<panda::Panda> new_panda) : tx
 	this->panda->set_esp_power(FALSE);
 	this->panda->set_safety_mode(panda::SAFETY_ALLOUTPUT);
 	this->panda->set_can_loopback(FALSE);
-	this->panda->set_alt_setting(1);
+	this->panda->set_alt_setting(0);
```

Comment:
Couldn't get enough throughput in interrupt mode (full speed USB max is 64 bytes per Ms)